examples/upload_storage_files_parallel.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3.7
+
+"""Upload files to Google Cloud Storage using asynchronous calls"""
+
+from typing import List, Dict, Tuple
+import asyncio
+
+from aiogoogle import Aiogoogle
+from aiogoogle.auth.creds import ServiceAccountCreds
+
+creds = ServiceAccountCreds(
+    scopes=[
+        "https://www.googleapis.com/auth/devstorage.read_only",
+        "https://www.googleapis.com/auth/devstorage.read_write",
+        "https://www.googleapis.com/auth/devstorage.full_control",
+        "https://www.googleapis.com/auth/cloud-platform.read-only",
+        "https://www.googleapis.com/auth/cloud-platform",
+    ],
+)
+
+aiogoogle_factory = Aiogoogle(service_account_creds=creds)
+
+
+async def get_creds():
+    """Set up the credentials for aiogoogle"""
+    await aiogoogle_factory.service_account_manager.detect_default_creds_source()
+
+
+asyncio.run(get_creds())
+
+
+async def async_gcs(file_list: List[Tuple[str, str, str]], bucket: str):
+    async with aiogoogle_factory:
+        storage = await aiogoogle_factory.discover("storage", "v1")
+        requests = []
+        for file_name, file_path, mime_type in file_list:
+            req = storage.objects.insert(bucket=bucket, name=file_name, upload_file=file_path)
+            # add the mime_type to the headers. You can autodetect mimetypes using python's
+            # built in `mimetypes` library
+            req.headers = {'Content-Type': mime_type}
+            # the upload endpoint for GCS is special, hence the replace
+            req.uri = req.url.replace('storage.googleapis.com/storage',
```

Comment:
This is the line I'm referring to

Reply:
I removed it (I amended the commit by accident so it's still a single commit).